contracts/MultiCreateV1.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity ^0.8.7;
+import "./interfaces/IControllerV1.sol";
+import "./interfaces/IMemberToken.sol";
+
+/* solhint-disable indent */
+
+contract MultiCreateV1 {
+    IMemberToken immutable memberToken;
+
+    constructor(address _memberToken) {
+        memberToken = IMemberToken(_memberToken);
+    }
+
+    function createPods(
+        IControllerV1 _controller,
+        address[][] memory _members,
+        uint256[] calldata _thresholds,
+        address[] memory _admins,
+        bytes32[] memory _labels,
+        string[] memory _ensStrings,
+        string[] memory _imageUrls
+    ) public returns (address[] memory) {
+        uint256 numPods = _thresholds.length;
+        require(_members.length == numPods, "incorrect members array");
+        require(_labels.length == numPods, "incorrect labels array");
+        require(_ensStrings.length == numPods, "incorrect ensStrings array");
+        require(_imageUrls.length == numPods, "incorrect imageUrls array");
+
+        uint256 nextPodId = memberToken.getNextAvailablePodId();
+
+        // 1 indexing to avoid relying on address(0)
```

Comment:
I don't understand how 1 indexing avoids relying on address(0)? I'm not quite sure what is happening in general. I think this function needs a docblock to explain the usage and expectations.

Reply:
Actually I do remember us talking about this, so I sort of remember what's going on here, but it should still be documented in code.